hooks/terraform_docs.sh

```
@@ -240,10 +240,19 @@
-    # shellcheck disable=SC2086
-    terraform-docs --output-mode="$output_mode" --output-file="$output_file" $tf_docs_formatter "$config_options" $args ./ > /dev/null
+
+    local tfdocs_cmd=(
+      terraform-docs
+        --output-mode="$output_mode"
+        --output-file="$output_file"
+        $tf_docs_formatter
+        $args
+    )
+    if [[ $have_config_flag == true ]]; then
+      ${tfdocs_cmd[@]} "--config=$config_file" ./ > /dev/null
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)